app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  def update_counter
+    user.update(posts_counter: user.posts_counter + 1)
+  end
+
+  def recent_comments()
+    comments.last(3)
+  end
```

Comment:
OPTIONAL

- [ ] You can use the  [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the #update_post_counter method every time a new post has been saved to the database, then make the `update_post_counter` method private